dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -39,7 +39,9 @@ startsecs=0
 command=/usr/sbin/fancontrol
 priority=4
 autostart=false
-autorestart=false
+autorestart=unexpected
+startsecs=10
+stopwaitsecs=10
```

Comment:
10 seconds is the default value for `startsecs` in supervisor, thus this line is unnecessary. However, I am fine with adding the line for the sake of making it explicit, in order to prevent any issues if the supervisor folks change to the default values in the future. However, I would suggest doing it in a separate PR, and adding the line to all relevant files in all applicable places at the same time.